src/QueryLogger.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Muffin\Webservice;
+
+class QueryLogger extends \Cake\Database\Log\QueryLogger
```

Comment:
I've been trying to implement this in https://github.com/UseMuffin/Webservice/pull/4/files#diff-b64b2c95409c473bd7333d6ff7f345c6R157, https://github.com/UseMuffin/Webservice/pull/4/files#diff-b64b2c95409c473bd7333d6ff7f345c6R91 and https://github.com/UseMuffin/Webservice/pull/4/files#diff-4fc7f66eb2d754bc065c57a73c800f5aR98

Just me know what you think.